osbs/utils.py

```diff
@@ -316,18 +318,19 @@ def make_name_from_git(repo, branch, limit=53, separator='-'):
 
     repo = git_repo_humanish_part_from_uri(repo)
     branch = branch or 'unknown'
+    hash_size = 5
 
     repo_chars = []
     branch_chars = []
     groups = ((repo, repo_chars), (branch, branch_chars))
 
-    size = len(separator)
+    size = sep_size = len(separator)
```

Comment:
This should just be updated to reflect the minimum `size`:
```python
size = (2 * len(separator)) + hash_size
```

Doing so will also avoid having to determine the limit on every loop iteration.